client/src/components/Filter.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../stylesheets/layout/_filter.scss';
+
+class Filter extends React.Component {
+  render() {
+    return (
+      <aside className="App--filter">
+        <div className="App--filter__option shown">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>filtros</p> </div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>humanos famosos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>comida</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>animales famosos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>nombres compuestos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>épicos</p></div>
+        <div className="App--filter__option hidden">
+          <input type="checkbox" className="switch" id="switch" />
+          <label for="switch"></label><p>clásicos</p></div>

```

Comment:
I'm sure this can be done with a loop instead of repeating so much code.

Reply:
you are right!

Reply:
we checked that SASSand Tailwind could work together and we installed it. Then I forgot to use it   :| my bad! i will improve it

Reply:
Perfect, but consider why do you need sass. Maybe it's not necesary ;)

Reply:
okey perfect, I'm gonna try to make a loop with it :)